### PR TITLE
Progress updates

### PR DESCRIPTION
**Pull request by pajowu**
- calculate task progress in backend
- show progress of conversion to wav

### CODE REVIEW

File: app/src/state/transcribe.ts [hunk 1]
```diff
@@ -65,8 +65,19 @@ export const startTranscription = createAsyncThunk<
...
   dispatch(openTranscribing());
 
+  dispatch(setProgress(0));
+  dispatch(setState(TranscriptionState.CONVERTING));
+
   const fileName = basename(path);
-  const wavFileContent = await convertToWav(path);
+  let wavFileContent;
+  try {
+    wavFileContent = await convertToWav(path, (p) => dispatch(setProgress(p)));
```
Comment (anuejn) [inline app/src/state/transcribe.ts:74]: I think it is somewhat bad ux to restart the progressbar from 0. maybe we add old-school multiprogressbars?
or we just allocate 10% of the progress to the convert to wav?

**Review by anuejn — CHANGES_REQUESTED** [review]
nice work :)
it would be nice to have the progressbar to be continous.
nit: there is a typo in the commit message ;)

File: app/src/state/transcribe.ts (hunk 1 above)
Comment (pajowu) [reply app/src/state/transcribe.ts:74]: I also think multi-progress-bars would be the best way. But maybe let's add them in the next step?

Comment (anuejn) [reply app/src/state/transcribe.ts:74]: sounds like a plan :)

Comment (pajowu) [reply app/src/state/transcribe.ts:74]: I created #240 for that.

**Comment by pajowu** [comment]
Fixed the typo

**Review by anuejn — APPROVED** [review]

**Pull request merged**